go.mod

```diff
@@ -5,13 +5,13 @@ go 1.17
 require (
```

Comment:
Maybe we should put the name of upgraded libs into the PR name?

Reply:
i think in body PR will better, do u?

Reply:
Ok, that will do, I think

Reply:
done